grunt/config/javascript.js

```diff
@@ -19,6 +19,17 @@ module.exports = function(grunt, options) {
           return grunt.config('helpers').includedFilter(filepath);
         },
         umdImports: [
+          '../node_modules/object.assign/dist/browser.js',
+          '../node_modules/react/umd/react.development.js',
+          '../node_modules/react-dom/umd/react-dom.development.js',
+          '../node_modules/html-react-parser/dist/html-react-parser.min.js'
+        ],
+        reactTemplates: [
+          '<%= sourcedir %>core/templates/**/*.jsx',
+          '<%= sourcedir %>components/*/templates/**/*.jsx',
+          '<%= sourcedir %>extensions/*/templates/**/*.jsx',
+          '<%= sourcedir %>menu/*/templates/**/*.jsx',
+          '<%= sourcedir %>theme/*/templates/**/*.jsx'
```

Comment:
Could these all be replaced with one `<%= sourcedir %>*/templates/**/*.jsx`?

Reply:
No, they're done like that to make them override in load order.